tests/testthat/test_ProcessingStep.R

```diff
@@ -15,3 +15,20 @@ test_that("ProcessingStep executeProcessingStep", {
     ## Pass optional arguments to ...
     expect_identical(executeProcessingStep(ps, na.rm = TRUE), 10)
 })
+
+test_that(".cat_fun works", {
+    res <- .cat_fun("sum")
+    expect_equal(res, "sum")
+    res <- .cat_fun(sum)
+    expect_equal(res, "user-provided function")
+})
+
+test_that("show,ProcessingStep works", {
+    ps <- ProcessingStep(sum, list(1:4))
+    show(ps)
```

Comment:
That's not a real test. This just fails if there is an error. What about `expect_output`?

Reply:
:) you got me - I was just too lazy... I will fix.